index.js

```diff
@@ -1,49 +1,63 @@
-const section = document.getElementById('book-section');
-const submit = document.getElementById('submit');
-const titleInput = document.getElementById('title-input');
-const authorInput = document.getElementById('author-input');
-
-let DATA = JSON.parse(localStorage.getItem('data')) || [];
-
-function add(author, title) {
-  DATA.push({ author, title });
-  localStorage.setItem('data', JSON.stringify(DATA));
-}
-
-function remove(author, title) {
-  DATA = DATA.filter((b) => b.author !== author || b.title !== title);
-  localStorage.setItem('data', JSON.stringify(DATA));
-}
-
-function show() {
-  section.innerHTML = '';
-  DATA.forEach((DATA) => {
-    const div = document.createElement('div');
-    const authorName = document.createElement('p');
-    authorName.textContent = `${DATA.author}`;
-    const titleName = document.createElement('p');
-    titleName.textContent = `${DATA.title}`;
-    const removeBtn = document.createElement('button');
-    removeBtn.textContent = 'Remove';
-    const hr = document.createElement('hr');
-
-    div.appendChild(authorName);
-    div.appendChild(titleName);
-    div.appendChild(removeBtn);
-    div.appendChild(hr);
-
-    section.appendChild(div);
-
-    removeBtn.addEventListener('click', () => {
-      remove(DATA.author, DATA.title);
-      show();
+class Book {
+  constructor() {
+    this.section = document.getElementById('book-section');
+    this.form = document.querySelector('form');
+    this.titleInput = document.getElementById('title-input');
+    this.authorInput = document.getElementById('author-input');
+    this.DATA = JSON.parse(localStorage.getItem('data')) || [];
+    this.counter = 0;
+    if (this.DATA.length > 0) {
+      this.show();
+    }
+  }
+
+  add(author, title) {
+    this.DATA.push({ author, title });
+    localStorage.setItem('data', JSON.stringify(this.DATA));
+  }
+
+  remove(author, title) {
```

Comment:
- [ OPTIONAL]
You can use unique id property for each book since it would make it easier to remove books with duplicate values. See example code below:

```javascript
  removeBook(bookID) {
    this.books = this.books.filter(({ id }) => bookID !== id); 
    return this.books; 
  }
```

Reply:
@t-yanick 
Thanks a lot for the kind review and suggestion. 😄